python/src/aiconfig/eval/common.py

```diff
@@ -127,3 +145,21 @@ def check_value_range(cls, values: dict[str, Any]) -> dict[str, Any]:
             )
         else:
             return values
+
+
+class TextRatingsData(cu.Record):
+    conciseness_rating: int
+    conciseness_confidence: float
+    conciseness_reasoning: str
+
+
+def get_llm_structured_response(
+    input_text: str,
+    chat_completion_create: CompletionTextToSerializedJSON,
```

Comment:
nit, should this be `text_completion_create` instead? Chat seems pretty specific

Reply:
I'll just tackle a bunch of these easy things together if that's ok (https://docs.google.com/document/d/15gx4vVmKVgI5bDHjSDd8xCQmTGcC8RrtLU0iCa32AIQ/edit)

Reply:
@rholinshead , after looking again, this is supposed to be an interface to generalize `openai.chat.completions.create` but _not_ necessarily to be more general than chat. The idea here was to allow mocking the openai function. So unless we think there's a reason otherwise, I think it makes sense to keep it as is.